server.js

```diff
@@ -12,6 +12,21 @@ var PORT = Number(argv.p) || Number(process.env.SCC_STATE_SERVER_PORT) || DEFAUL
 var CLUSTER_SCALE_DELAY = Number(argv.d) || Number(process.env.SCC_STATE_SERVER_SCALE_DELAY) || DEFAULT_CLUSTER_SCALE_DELAY;
 var AUTH_KEY = process.env.SCC_AUTH_KEY || null;
 var FORWARDED_FOR_HEADER = process.env.FORWARDED_FOR_HEADER || null;
+/**
+ * Log levels:
+ * 3 - log everything
+ * 2 - warnings and errors
+ * 1 - errors only
+ * 0 - log nothing
+ */
+var LOG_LEVEL;
+if (typeof argv.l !== 'undefined') {
+  LOG_LEVEL = Number(argv.l);
+} else if (typeof process.env.SCC_STATE_LOG_LEVEL !== 'undefined') {
+  LOG_LEVEL = Number(process.env.SCC_STATE_LOG_LEVEL);
+} else {
+  LOG_LEVEL = 1;
+}
```

Comment:
@MegaGM The default LOG_LEVEL should be 3 in this case (only for scc-state). The reason is that the messages logged by scc-state are quite important because they give information about the state of the cluster.

Also the logging is fairly minimal anyway.